apps/wallet/src/ui/app/components/menu/content/PasswordInputDialog.tsx

```diff
@@ -63,21 +71,36 @@ export function PasswordInputDialog({
                                 name="password"
                             />
                         </FieldLabel>
+                        <div className="text-center mt-4">
+                            <Text
+                                variant="p2"
+                                color="steel-dark"
+                                weight="normal"
+                            >
+                                This is the password you currently use to lock
+                                and unlock your Sui wallet.
+                            </Text>
+                        </div>
                     </div>
                     <div className="flex flex-col flex-nowrap gap-3.75 self-stretch">
                         <Button
                             type="submit"
                             variant="primary"
                             size="tall"
-                            text="Continue"
+                            text={continueLabel}
                             loading={isSubmitting}
                             disabled={!isValid}
+                            after={finalStep ? null : <ArrowRight16 />}
```

Comment:
I'd probably make the argument here to leave the default Continue -> button instead of adding complexity to this component. I still think it makes sense in context since we have the `Import Account` header at the top. 

![image](https://user-images.githubusercontent.com/122397493/221078521-88e3dbf6-8686-477d-9809-d443f3aad276.png)

If we want to leave this as-is, I think accepting `after` as a prop or renaming `finalStep` to something more explicit like `showArrowIcon` would be more clear

Reply:
renamed it to showArrowIcon